feature-utils/poly-look/src/react-components/overlays/sideSheet.css

```diff
@@ -0,0 +1,34 @@
+.poly-side-sheet {
+  max-width: 288px;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: var(--color-gray-100);
+  color: var(--color-dark);
+  z-index: 100;
+  overflow: hidden scroll;
+  padding: 18px 16px 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+}
+.poly-side-sheet .header {
+  display: flex;
+  align-items: center;
+  min-height: 48px;
+  margin-bottom: 23px;
```

Comment:
I seem to remember margins needed to be a multiple of 4?

> Maybe we should have, if not a root CSS, at least a set of CSS guidelines

Reply:
There's some style guides in Figma for this - that's true indeed, we have a 4px grid. BUT: Texts usually need to be moved down 1px to look "right", IIRC, so maybe this ends up looking just right this way. Let's see if the designers complain, they have eagle eyes.

Reply:
Well, I've removed this, for now. Maybe I wasn't using Figma right, but 23px was what it was showing me. I think it might make a comeback when I'll start using this component if I find myself constantly adding margin-top to the content.